src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmissingwhitespaceafterasterisk/InputJavadocMissingWhitespaceAfterAsterisk.java

```diff
@@ -0,0 +1,59 @@
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocmissingwhitespaceafterasterisk;
+class InputJavadocMissingWhitespaceAfterAsterisk
+{
+    /****
+     *
+     * This is ok.
+     ****/
+    void foo() {}
+
+    /**
+     *  @see See somthing.
+     */
+    void foo1() {}
+
+    /**
+     *This is not ok.
+     **/
+    void foo2() {}
+
+    /**
+     * This is not ok.
```

Comment:
This is confusing. Should it be
```
/**
 * This is ok.
 *@see violtaion
 */
```
? Or maybe `There is a violation on the next line` ?

Reply:
done